config/secrets.yml

```diff
@@ -29,13 +29,13 @@ test:
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
-  saml_issuer: urn:gov:gsa:SAML:2.0.profiles:sp:sso:rails-<%= ENV["SAML_ENV"] %>
-  idp_sso_url: https://idp.<%= ENV["SAML_ENV"] %>.login.gov/api/saml/auth
-  idp_slo_url: https://idp.<%= ENV["SAML_ENV"] %>.login.gov/api/saml/logout
-  http_auth_username: <%= ENV["SP_NAME"] %>
-  http_auth_password: <%= ENV["SP_PASS"] %>
+  secret_key_base: <%= ENV.fetch("SECRET_KEY_BASE") %>
+  saml_issuer: urn:gov:gsa:SAML:2.0.profiles:sp:sso:rails-<%= LoginGov::Hostdata.env %>
+  idp_sso_url: https://idp.<%= LoginGov::Hostdata.env %>.<%= LoginGov::Hostdata.domain %>/api/saml/auth
+  idp_slo_url: https://idp.<%= LoginGov::Hostdata.env %>.<%= LoginGov::Hostdata.domain %>/api/saml/logout
+  http_auth_username: <%= ENV.fetch("SP_NAME") %>
```

Comment:
Can we rename `SP_NAME` to something more descriptive while we're at it?

Reply:
Done!